### ISSUE TITLE
Add section on using Rust in Deno

### ISSUE THREAD

**Issue by dsherret** [denoland/docs#101]
We should add wasmbuild to the manual https://deno.com/blog/wasmbuild

I think this would go under the "Using WebAssembly" section.

**Comment by thisisjofrank** [comment]
Is this now covered by https://docs.deno.com/runtime/reference/wasm/ ? @dsherret

**Comment by dsherret** [comment]
Nope, not yet. This would be to add https://github.com/denoland/wasmbuild to the docs
